### ISSUE TITLE
How i can select only two columns of sub documents using populate 

### ISSUE THREAD

**Issue by osamabari** [edwardhotchkiss/mongoose-paginate#111]
Hi ,
 I am facing issue while getting sub documents  data from populate. i want to get only two columns instead of  holding whole object.

Is there any option like mongoose where we can select columns 

eg  ( populate:('collectionname',['column1',column2] )

**Comment by OnnoGabriel** [comment]
Yes, the style of the option is the same as in the mongoose ``.populate()`` function:

``populate: {path: 'fieldname', select: 'column1 column2'},``

(I was passing by looking for this very answer).
